README.md

```diff
@@ -11,6 +11,7 @@ This fairly simple monitoring plugin monitors the health of your galera cluster
     -P port [Default: 3306] Port
     -u username [Default: root] Username to monitor (Needs privilege to execute SHOW STATUS)
     -p password [Default: <empty>] Password
+    -n Don't use a password
```

Comment:
It would make more sense only to apply --password if -p has been set. Therefore -n would be obsolete.

Reply:
Well, it would be a breaking change, as an empty password is different from no password. I've updated the MR accordingly.